R/module-repository.R

```diff
@@ -310,14 +326,15 @@ setMethod(
       ua <- user_agent(getOption("spades.useragent"))
       pat <- Sys.getenv("GITHUB_PAT")
       request <- if (identical(pat, "")) {
-        GET(zip, ua, write_disk(localzip))
+        GET(zip, ua, write_disk(localzip, overwrite = overwrite))
       } else {
         message("Using GitHub PAT from envvar GITHUB_PAT")
-        GET(zip, ua, config = list(config(token = pat)), write_disk(localzip))
+        GET(zip, ua, config = list(config(token = pat)),
+            write_disk(localzip, overwrite = overwrite))
       }
       stop_for_status(request)
 
-      files <- unzip(localzip, exdir = file.path(path), overwrite = TRUE)
+      files <- unzip(localzip, exdir = file.path(path), overwrite = overwrite)
```

Comment:
this always needs to overwrite, even if `overwrite = FALSE`

Reply:
done